src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,116 @@
-export const Pagination = () => {};
+import React from 'react';
+import cn from 'classnames';
+
+interface Props {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  onPageChange: (page: number) => void;
+}
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  onPageChange,
+}) => {
+  const pagesAmount = Math.ceil(total / perPage);
+
+  const visibleItems = [];
+  const visiblePages = [];
+
+  for (let i = 1; i <= pagesAmount; i += 1) {
+    visiblePages.push(i);
+  }
```

Comment:
use `getNumbers` function, which you used for creating array of numbers, instead